src/main/java/racingcar/controller/GameController.java

```diff
@@ -0,0 +1,52 @@
+package racingcar.controller;
+
+import java.util.ArrayList;
+import java.util.List;
+import racingcar.domain.Car;
+import racingcar.domain.Cars;
+import racingcar.util.RandomNumberGenerator;
+import racingcar.util.RandomNumberMaker;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+public class GameController {
+    private final InputView inputView = new InputView();
+    private final OutputView outputView = new OutputView();
+    private final RandomNumberGenerator randomNumberGenerator = new RandomNumberMaker();
+
+    public void run() {
+        Cars cars = setCars();
```

Comment:
```suggestion
        Cars cars = createCars();
```
제가 너무 틀에 박힌 생각일 수도 있겠지만, set은 보통 값을 지정하는 의미로 사용해서 cars를 생성하는 것이기도 하니 create으로 변경하면 좋을 것 같다고 생각합니다!